programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1374,16 +1374,21 @@ static void enroll(int argc, char *argv[]) {
         exit(-1);
     }
 
+    #include <fcntl.h>
+    #include <errno.h>
     FILE *outfile;
-    if ((outfile = fopen(config_file, "wb")) == NULL) {
-        ZITI_LOG(ERROR, "failed to open file %s: %s(%d)", config_file, strerror(errno), errno);
-        exit(-1);
-
+    outfile = open(config_file, O_CREAT | O_WRONLY | O_EXCL, S_IRUSR | S_IWUSR);
+    if (outfile < 0) {
+        if (errno == EEXIST) {
+            ZITI_LOG(ERROR, "refusing to overwrite existing config file %s: %s(%d)", config_file, strerror(errno), errno);
+            exit(-1);
+        }
+    } else {
+        if ((outfile = fdopen(config_file, "wb")) == NULL) {
+            ZITI_LOG(ERROR, "failed to open file %s: %s(%d)", config_file, strerror(errno), errno);
+            exit(-1);
+        }
     }
-    uv_loop_t *l = uv_loop_new();
```

Comment:
Did you remove these 3 lines accidentally?

Reply:
@mary-dcouto You are correct! I've restored the unintentionally-dropped lines in the last commit.